zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -445,11 +445,11 @@ public void runAll() {
         AuthenticationInfo authenticationInfo = new AuthenticationInfo();
         authenticationInfo.setUser(cronExecutingUser);
         p.setAuthenticationInfo(authenticationInfo);
-
-        p.setListener(jobListenerFactory.getParagraphJobListener(this));
-        Interpreter intp = factory.getInterpreter(getId(), p.getRequiredReplName());
-
-        intp.getScheduler().submit(p);
+        try {
+          run(p.getId());
+        } catch (InterpreterException intpException) {
+          throw intpException;
+        }
```

Comment:
@bzz 
Thank you for your feedback.
I wanted to express the `Exception` section occurs.
try .. catch means there is no action.
Does this part adjusts the confusion? If you have a problem, I would remove the `try..catch`.